cmd/sfu/main.go

```diff
@@ -85,7 +88,7 @@ func main() {
 	case "panic":
 		logrus.SetLevel(logrus.PanicLevel)
 	default:
-		logrus.SetLevel(logrus.InfoLevel)
+        logrus.Fatalf("unrecognised log level: %s", config.LogLevel)
```

Comment:
It probably would fail due to the formatting :)

Reply:
ah yep, that's what I get for editing in vim rather than opening up an IDE.